### ISSUE TITLE
netty-transport-native-epoll-VERSION-linux-x86_64.jar isn't an OSGi bundle

### ISSUE THREAD

**Issue by almibe** [netty/netty#3781]
Hello, I've noticed that the linux-x86_64 versions of the netty-transport-native-epoll jar isn't an OSGi bundle while the regular jars are.  This seems to be the case for all versions of this jar.

**Comment by Scottmitch** [comment]
@almibe - Thanks for reaching out.  Do you have any interest in submitting a PR to resolve this issue?  We love PRs :)

**Comment by almibe** [comment]
I like making pull requests but I haven't been able to make netty build at all.  If I check out netty and do "mvn install" shouldn't that work?

**Comment by Scottmitch** [comment]
> If I check out netty and do "mvn install" shouldn't that work?

Yes. If you provide more details perhaps we can help.  Make sure you are using a "recent" version of maven.  See http://netty.io/wiki/setting-up-development-environment.html for details.

**Comment by normanmaurer** [comment]
@almibe did you ever manage to find time ?

**Comment by normanmaurer** [comment]
@almibe ping

**Comment by almibe** [comment]
Sorry for some reason I didn't see the notifications on this issue.  I only ran into this issue with ratpack and since filing this issue I've started using vertx-web which doesn't use this feature of netty.  I've also started building "fat bundles" in OSGi which avoids this issue entirely so I'm no longer having this issue.  I'll close this since I can't really test it anymore and if someone else wants netty-transport-native-epoll-VERSION-linux-x86_64.jar to be an OSGi bundle they can open an issue.  Sorry for my late response.
